woocommerce/woocommerce
# Revert "[Enhancement]: Add option to show price on the place order button"

### Changes proposed in this Pull Request:

Reverts #49252 - this is because the implementation of the separator is not ideal and could introduce a customisation path for developers and merchants that would be hard to support going forward. Rather than introducing this, we would prefer to refine the approach.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to the Checkout page in the editor
2. Highlight the Checkout Actions block and ensure no "Show price in the button" option is available in the settings sidebar.
3. Add an item to your cart and view the Checkout block on the front end. Ensure the button renders correctly and that clicking it shows the spinner and places the order.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
